build-system/tasks/test-report-upload.js

```diff
@@ -63,24 +63,25 @@ async function getReport(testType) {
  * @return {Object.<string,Object>} Object containing the build, job, and test results.
  */
 function addJobAndBuildInfo(testType, reportJson) {
-  const buildNumber = ciBuildNumber();
+  const buildId = ciBuildId();
   const commitSha = ciCommitSha();
-  const jobNumber = ciJobNumber();
+  const jobId = ciJobId();
 
-  if (!buildNumber || !commitSha || !jobNumber) {
+  if (!buildId || !commitSha || !jobId) {
     throw new ReferenceError('CI fields are not defined.');
   }
 
+  // (TODO ampproject/amp-github-apps/pull:1194) Update field names in database.
   return {
     repository: process.env.GITHUB_REPOSITORY,
     results: reportJson,
     build: {
-      buildNumber,
+      buildNumber: buildId,
```

Comment:
I would suggest the opposite of this. Let me throw together a quick PR converting the field name on the backend handler, and you can report buildId and jobId here.

Reply:
Even better. Thanks for the help.

Reply:
See here: https://github.com/ampproject/amp-github-apps/pull/1195
```suggestion
      buildId,
```